semcore/input-tags/src/InputTags.tsx

```diff
@@ -17,8 +17,9 @@ export interface IInputTagsProps extends Omit<IInputProps, 'size'>, IScrollAreaP
    * @default m
    */
   size?: InputTagsSize;
-  /** @deprecated use `onAppend` instead */
-  /** Event is called when tag needs to be added */
+  /** Event is called when tag needs to be added
+   * @deprecated use `onAppend` instead
```

Comment:
Why this file was affected?

Reply:
I just walked near this file and noticed that the description of the prop was written incorrectly I mean users don't see that it was deprecated and need to use another function
<img width="979" alt="Screenshot 2022-09-15 at 18 30 04" src="https://user-images.githubusercontent.com/44463016/190458126-18040ebe-d4ac-4fda-8221-6a04e60dcd14.png">

Reply:
I just fixed description

Reply:
<img width="955" alt="Screenshot 2022-09-15 at 18 31 28" src="https://user-images.githubusercontent.com/44463016/190458409-db2da3cd-5d58-402e-9d9f-b9c4e5e4dfa6.png">

Reply:
but yes I understand thta in this case commit message and these changes are not the same. anyway I can return everything as it was :)